### PR TITLE
bump fluentd-gcp version, apiVersion and other config changed

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
bump fluentd-gcp version, apiVersion and other config changed

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE

```

### CODE REVIEW

**Comment by coffeepac** [comment]
looks good to me, but I can't approve this.

thanks @zouyee !

**Comment by roberthbailey** [comment]
Will let @crassirostris / @piosz add the lgtm label once they have reviewed.

**Comment by crassirostris** [comment]
Please fix tests. Currently it seems some changes made agents unschedulable

**Comment by crassirostris** [comment]
Thanks a lot for your contribution!

File: cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml [hunk 1]
```diff
@@ -103,12 +103,17 @@ spec:
...
         beta.kubernetes.io/fluentd-ds-ready: "true"
       tolerations:
+      # Allow the pod to run on the master.  This is required for
+      # the master to communicate with pods.
       - key: "node.alpha.kubernetes.io/ismaster"
         effect: "NoSchedule"
-      - operator: "Exists"
-        effect: "NoExecute"
-      - operator: "Exists"
+      - operator: NoExecute
```
Comment (crassirostris) [inline cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml]: Are you sure this is correct? You're saying there's a bug currently?

**Comment by crassirostris** [comment]
From the [test log](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/57895/pull-kubernetes-e2e-gce/15693/#sig-instrumentation-cluster-level-logging-implemented-by-stackdriver-should-ingest-events):

```
node e2e-15693-minion-group-1g3q doesn't have logging agents, want 1
```

Please fix

**Pull request closed without merging**